lib/enkf/tests/enkf_forward_init_FIELD.cpp

```diff
@@ -46,15 +46,43 @@ void create_runpath(enkf_main_type * enkf_main, int iter) {
 
 
 
+void install_file( const ecl::util::TestArea& ta, const char * input_src_file ) {
+  if (util_is_abs_path( input_src_file))
+    return;
+  else {
+    std::string src_file = ta.original_path(input_src_file);
+    char * src_path;
+
+    util_alloc_file_components(input_src_file, &src_path, NULL, NULL);
+
+    if (!util_entry_exists( src_path ))
+      util_make_path( src_path );
+
+    if (util_file_exists( src_file.c_str() )) {
+      char * target_file   = util_alloc_filename( ta.test_cwd().c_str(), input_src_file, NULL );
+      util_copy_file( src_file.c_str() , target_file );
+      free( target_file );
+    }
+
+    free(src_path);
+  }
+}
```

Comment:
I have a feeling this was already discussed somewhere, but why `install_file` is not a method in `TestArea`?

Reply:
Hehe; there was a discussion at some point: https://github.com/equinor/libecl/pull/583 - I lost that argument ...